### ISSUE TITLE
Background and Foreground colors modified

### ISSUE THREAD

**Issue by marcelo-es** [projekt0n/github-theme-contrib#11]
Hi~ I thanks for putting this project together, it's super tidy!

But I noticed that the background colours were not matching the default canvas colours of GitHub, so found that they are actually changed manually in [palettes.ts](https://github.com/projekt0n/github-theme-contrib/blob/4eb8179ec897b6f3e658fe67acf60d1c5a91c35d/src/lib/palettes.ts#L11).

```typescript
dark.canvas.default = dark.scale.gray[6];
dark.fg.default = "#e6edf3";

dark_colorblind.canvas.default = dark_colorblind.scale.gray[9];
dark_colorblind.fg.default = dark_colorblind.scale.gray[1];

dark_dimmed.canvas.default = dark_dimmed.scale.gray[9];
dark_dimmed.fg.default = dark_dimmed.scale.gray[1];

dark_high_contrast.canvas.default = dark_high_contrast.scale.gray[9];
dark_high_contrast.fg.default = dark_high_contrast.scale.gray[1];

dark_tritanopia.canvas.default = dark_tritanopia.scale.gray[9];
dark_tritanopia.fg.default = dark_tritanopia.scale.gray[1];

light.canvas.default = light.scale.white;
light.fg.default = light.scale.black;

light_colorblind.canvas.default = light_colorblind.scale.white;
light_colorblind.fg.default = light_colorblind.scale.black;

light_high_contrast.canvas.default = light_high_contrast.scale.white;
light_high_contrast.fg.default = light_high_contrast.scale.black;

light_tritanopia.canvas.default = light_tritanopia.scale.white;
light_tritanopia.fg.default = light_tritanopia.scale.black;
```

Was wondering on what the rationale is? Wouldn't it be better to follow the GitHub original colours?

Thanks!

**Comment by ful1e5** [comment]
The colors provided by Primer are meant specifically for the GitHub web platform and its web products. However, this doesn't apply to the popular [github-vscode-theme](https://github.com/primer/github-vscode-theme), which is widely used. Developers often adjust and test colors before incorporating changes into Primer.

When looking at the VSCode theme, you'll observe that colors are modified within the [color.js](https://github.com/primer/github-vscode-theme/blob/38b50d41bdc6b190b7351e220956af043221d763/src/colors.js#L11C3-L46C4) file for both the 'light' and 'dark' versions. There are more noticeable differences in colors between Primer and the VSCode theme.

In the past, I encountered issues with the [github-nvim-theme](https://github.com/projekt0n/github-nvim-theme), as it appeared distinct from the VSCode theme. To address this, I integrated these mentioned overrides into the Neovim theme to align with the VSCode theme.

Furthermore, I've incorporated these overrides here to **ensure consistency between the terminal themes and the neovim theme**, as many users use both simultaneously.

I hope this explanation clarifies the situation.
